events/guildMemberAdd.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         const embed = new EmbedBuilder()
             .setColor(CORNFLOWER_BLUE)
             .setTitle(`Nouveau membre`)
-            .setDescription(`<@${member.id}>`)
+            .setDescription(member)
```

Comment:
Il faut que ce soit une string

```suggestion
            .setDescription(`${member}`)
```

Reply:
Pareil

```suggestion
            .setDescription(member.toString())
```